commons/loom/src/main/java/org/infinispan/commons/jdkspecific/JREVersion.java

```diff
@@ -0,0 +1,9 @@
+package org.infinispan.commons.jdkspecific;
+
+public class JREVersion {
+   private JREVersion() { }
+
+   public static boolean isJava15OrNewer() {
```

Comment:
Well the idea was originally that I would detect if a thread group produces non blocking threads that it can't be configured with a blocking executor. But in Loom we will want this. How do you think we should do this then?

Reply:
I can just remove this for now though I guess. And we can deal with it later.

Reply:
I would have been fine with something like `isLoomAvailable()`, or `virtualThreadsAvailable()`, I didn't want `isJava15OrNewer()` because stock OpenJDK 15 still doesn't have Loom.

Reply:
Oh okay, I misunderstood what you were saying. Either way I removed it for now.

Reply:
I have created https://issues.redhat.com/browse/ISPN-11563 which we can look into later about this.